CRM/Financial/DAO/PaymentProcessor.php

```diff
@@ -185,7 +185,7 @@ public function __construct() {
    */
   public static function getReferenceColumns() {
     if (!isset(Civi::$statics[__CLASS__]['links'])) {
-      Civi::$statics[__CLASS__]['links'] = static::createReferenceColumns(__CLASS__);
+      Civi::$statics[__CLASS__]['links'] = static ::createReferenceColumns(__CLASS__);
```

Comment:
@eileenmcnaughton this seems off

Reply:
Yeah gencode didn't pick up the changes? I've fixed it manually for now